src/components/Article.jsx

```diff
@@ -80,15 +80,11 @@ const Article = () => {
               objectFit: 'cover',
             }}
           />
-          {article.body &&
-            article.body.map((paragraph) => (
-              <>
-                <p data-cy='article-body' className='article-body'>
-                  {paragraph}
-                </p>
-                <br />
-              </>
-            ))}
+          {article.body && (
+            <p data-cy='article-body' className='article-body'>
+              {article.body}
+            </p>
+          )}
         </>
```

Comment:
I think that we will now get '\n\n' as part of the string from the API. 
We can use this to split the text directly here in the render and display something siilar to above with the `<br />`

Reply:
Or how do we breka the paragrpahs now? :)

Reply:
we handle it in css. `white-space: pre-wrap;` will transfoen \n into line breat and \t into tabs

Reply:
we put a style in css that breakes the lines when you say \n so you can continue with that maybeee, br is not a good practice i think

Reply:
lolz

Reply:
Could put a picture in dere to prove :3

Reply:
What's a \t